### PR TITLE
es6-classes - Marianne Cuevas

### PR DESCRIPTION
**Pull request by mariannecuevas**

### CODE REVIEW

File: es6-classes/circle.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+/* eslint-disable no-undef -- Shape is imported by .html */
+/* exported Circle */
+class Circle extends Shape {
+  constructor(radius) {
+    super();
+    this.radius = radius;
+    this.area = radius * Math.PI;
+    this.circumference = 2 * radius * Math.PI;
```
Comment (scrumptiousdev) [inline es6-classes/circle.js]: So, we will have to "override" Shape's constructor values (area and circumference) from Circle. How do we do that?

File: es6-classes/square.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+/* eslint-disable no-undef -- Shape is imported by .html */
+/* exported Square */
+class Square extends Shape {
+  constructor(width) {
+    super();
+    this.width = width;
+    this.area = width * width;
+    this.circumference = width * 4;
```
Comment (scrumptiousdev) [inline es6-classes/square.js]: So, we will have to "override" Shape's constructor values (area and circumference) from Square. How do we do that?

File: es6-classes/square.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+  }
+
+  print() {
+    return 'The area is ' + this.area + '. ' + 'The circumference is ' + this.circumference + '. ' + 'The width is ' + this.width + '.';
```
Comment (scrumptiousdev) [inline es6-classes/square.js]: Also override the print method to "print the Shape's message" followed by the "circle's radius". So.. we are not overriding the whole thing but we need to use Shape's print method as well

This goes same for Circle

**Review by scrumptiousdev — CHANGES_REQUESTED** [review]

File: es6-classes/circle.js (hunk 1 above)
Comment (mariannecuevas) [reply es6-classes/circle.js]: I tried different ways, but I'm not sure which is the correct or most efficient way to do it.

Comment (scrumptiousdev) [reply es6-classes/circle.js]: Looks great! :)

**Review by scrumptiousdev — APPROVED** [review]
Approved! :)

**Pull request merged**